tcms/utils/tests/test_github.py

```diff
@@ -27,3 +36,13 @@ def test_invalid_signature_header(self):
 
         self.assertIsInstance(result, HttpResponseForbidden)
         self.assertEqual(HTTPStatus.FORBIDDEN, result.status_code)
+
+    def test_valid_signature_header(self):
+        request = self.factory.post(self.url)
+        sha1 = hmac.new(key=b'secret',
+                        msg=request.body,
+                        digestmod=hashlib.sha1).hexdigest()
+        request.META['HTTP_X_HUB_SIGNATURE'] = f'sha1={sha1}'
```

Comment:
can be replaced with github.calculate_signature() b/c here we are asserting on the truthful result of the verify function